.github/workflows/main.yml

```diff
@@ -34,11 +34,38 @@ jobs:
         run: |
           wget --no-verbose http://www.crc.nd.edu/~kmohrman/files/root_files/for_ci/ttHJet_UL17_R1B14_NAOD-00000_10194.root
 
-      - name: Run processors over test files with futures executor
+      - name: Pytest setup
         run: |
           conda install -y -n coffea-env -c conda-forge pytest pytest-cov root_base
           mkdir histos
-          conda run -n coffea-env pytest --cov=./ --cov-report=xml -rP
+
+      - name: Test unit
+        run: |
+          conda run -n coffea-env pytest --cov=./ --cov-report=xml -rP tests/test_unit.py
+
+      - name: Test histEFT
+        run: |
+          conda run -n coffea-env pytest tests/test_HistEFT_add.py
+
+      - name: Test utils
+        run: |
+          conda run -n coffea-env pytest tests/test_utils.py
+
+      - name: Test update json
+        run: |
+          conda run -n coffea-env pytest tests/test_update_json.py
+
+      - name: Test make 1d quad plots
+        run: |
+          conda run -n coffea-env pytest tests/test_make_1d_quad_plots.py 
+
+      - name: Run processors over test files with futures executor
+        run: |
+          conda run -n coffea-env pytest tests/test_futures.py
+
+      - name: Run processors over test files with workqueue executor
+        run: |
+          conda run -n coffea-env pytest tests/test_workqueue.py
```

Comment:
I think once we append `--cov=./ --cov-report=xml -rP --cov-append` to these test, this PR should be good to go.